internal/datanode/flow_graph_time_tick_node.go

```diff
@@ -48,8 +50,17 @@ type ttNode struct {
 	BaseNode
 	vChannelName   string
 	channel        Channel
-	lastUpdateTime time.Time
+	lastUpdateTime *atomic.Time
 	dataCoord      types.DataCoord
+
+	updateCPLock  sync.Mutex
+	notifyChannel chan checkPoint
+	closeChannel  chan bool
```

Comment:
Please change it to `chan struct{}` for closeChannel

Reply:
updated